apps/meteor/app/federation-v2/client/Federation.ts

```diff
@@ -0,0 +1,16 @@
+import { ValueOf } from '@rocket.chat/core-typings';
+
+import { RoomMemberActions } from '../../../definition/IRoomTypeConfig';
+
+const allowedActionsInFederatedRooms: ValueOf<typeof RoomMemberActions>[] = [
+	RoomMemberActions.REMOVE_USER,
+	RoomMemberActions.INVITE,
+	RoomMemberActions.JOIN,
+	RoomMemberActions.LEAVE,
+];
+
+export class Federation {
+	public static federationActionAllowed(action: ValueOf<typeof RoomMemberActions>): boolean {
+		return allowedActionsInFederatedRooms.includes(action);
+	}
+}
```

Comment:
Let's avoid using classes as namespaces. Prefer functions in modules, as they can be imported like:

```ts
import * as Federation from './federation';

Federation.actionAllowed(action);
```